CHANGELOG.rst

```diff
@@ -12,7 +12,8 @@ Change Log
 Unreleased
 ~~~~~~~~~~
 
-*
+* Support for Python 3.5-3.8
```

Comment:
We can mention here that this supports Juniper

Reply:
Added!